synth.py

```diff
@@ -24,7 +24,14 @@
 
 gapic = gcp.GAPICMicrogenerator()
 versions = ['v2', 'v2beta1']
-for version in versions:
+default_version = 'v2'
+
+# Rearrange the default version to the last item in the array, to generate appropriate system-test
```

Comment:
This is too complicated. Can we just do `versions = ['v2beta1', 'v2']` and have a comment that the default version must come the last?

Reply:
I'll do it :)